capsules/src/test/aes_ccm.rs

```diff
@@ -34,7 +34,8 @@ impl<'a, A: AES128CCM<'a>> Test<'a, A> {
             tests: [
                 (
                     &BEACON_UNSECURED[0..26],
-                    &BEACON_UNSECURED[26..26],
+                    //&BEACON_UNSECURED[26..26],
+                    &[],
```

Comment:
```suggestion
                    &[],
```

I get why this was just commented, but I think it would be clearer to just be explicit that it's empty instead of implying that the array matters or that there's something reasonable to uncomment.